### PR TITLE
Add some information to the readme

### PR DESCRIPTION
**Pull request by chee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,38 @@
-# OCC
-the **o**rigami **c**omponent **c**onverter
+# Origami Component Converter
 
-## usage
+## What is this?
+
+This is a tool used during build process of Origami components, to prepare them
```
Comment (JakeChampion) [inline README.md]:
```suggestion
This is a tool used during the build process of Origami components, to prepare them for publishing to npm.
```

File: README.md [hunk 2]
```diff
@@ -1,16 +1,38 @@
...
+for publish on npm.
```
Comment (JakeChampion) [inline README.md]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,16 +1,37 @@
...
-## usage
+## What is this?
+
+This is a tool used during the build process of Origami components, to prepare them for publishing to npm.
+
+### how does it do that?
+
+It gets run in the project directory of a component.
+
+First, it converts the [bower.json](https://github.com/bower/spec/blob/59c8f0e8f8444cbdd71a091919d28d0fa29d56fe/json.md) to an npm-ready
```
Comment (JakeChampion) [inline README.md]:
```suggestion
First, it converts the [bower.json](https://github.com/bower/spec/blob/59c8f0e8f8444cbdd71a091919d28d0fa29d56fe/json.md) to an npm-ready [package.json](https://docs.npmjs.com/files/package.json) and writes it to disk.
```

File: README.md [hunk 4]
```diff
@@ -1,16 +1,37 @@
...
+[package.json](https://docs.npmjs.com/files/package.json) and writes it to disk.
```
Comment (JakeChampion) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,16 +1,37 @@
...
+
+* append `@financial-times/` to the name, to place it under our npm organisation
+* rewrite dependencies from the `bower.json` to their npm names
+* create a map of aliases for the upcoming compile step
+<!-- * merge with dependencies and scripts from any existing `package.json` -->
+
+Then, it compiles the javascript to commonjs using babel. The
```
Comment (JakeChampion) [inline README.md]:
```suggestion
Then, it compiles the javascript to commonjs using babel. The [babel-plugin-import-rewrite](./lib/babel-plugin-import-rewrite.js) plugin uses the generated aliases field to rewrite any dependencies whose names differ between bower and npm.
```

File: README.md [hunk 6]
```diff
@@ -1,16 +1,37 @@
...
+[babel-plugin-import-rewrite](./lib/babel-plugin-import-rewrite.js) plugin
```
Comment (JakeChampion) [inline README.md]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,16 +1,37 @@
...
+uses the generated aliases field to rewrite any dependencies whose names differ
```
Comment (JakeChampion) [inline README.md]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -1,16 +1,37 @@
...
+between bower and npm.
```
Comment (JakeChampion) [inline README.md]:
```suggestion
```

**Comment by chee** [comment]
@JakeChampion are you trying to say that it's Origami style not to wrap markdown at 80 columns?

**Comment by JakeChampion** [comment]
I don't think we have a style, I just assumed it was unintentional

**Comment by chee** [comment]
@JakeChampion it was on purpose, because then it looks nice even when rendered as plain text

i quite often read README.md in `$PAGER`, which does not wrap for everyone.

i'll accept your changes though because i don't have very strong feelings

**Pull request merged**